src/main/java/org/folio/rest/service/EventPublisher.java

```diff
@@ -61,6 +63,7 @@ private Event createEvent(EventType eventType, String payload) {
       .withEventMetadata(new EventMetadata()
         .withPublishedBy(PubSubClientUtils.getModuleId())
         .withTenantId(tenantId)
-        .withEventTTL(1));
+        .withEventTTL(1)
+        .withCreatedBy(okapiHeaders.get("source")));
```

Comment:
source value should be located in the payload not in the event metadata etc.

Reply:
I fixed the code, thank you.